src/Plugin.php

```diff
@@ -220,6 +246,27 @@ public static function register_acf() {
     ]);
   }
 
+  /**
+   * Loads scripts as deferred or async.
+   *
+   * @implements script_loader_tag
+   */
+  public static function script_loader_tag($tag) {
+    $loading_types = [];
+    $loading_types['defer'] = apply_filters(Plugin::L10N . '/scripts_load_defer', static::SCRIPTS_LOAD_DEFER);
+    $loading_types['async'] = apply_filters(Plugin::L10N . '/scripts_load_async', static::SCRIPTS_LOAD_ASYNC);
```

Comment:
This architecture is quite confusing. I first thought that these would be two different processing staging, but it turns out that they are just two separate stacks per attribute.  I would suggest the following instead:

1. Use the script handles instead of the filesystem paths, because the latter can change more often. The handle is passed to the filter; see https://developer.wordpress.org/reference/hooks/script_loader_tag/

1. Use only one constant to specify both async and defer. Use the array key for the handle, and explicit value items to adjust the attributes.

```php
  const SCRIPTS = [
    'wc-add-to-cart' => ['async' => FALSE, 'defer' => TRUE],
    'wc-add-to-cart-variation' => ['async' => FALSE, 'defer' => TRUE],
  ];
```